setup.cfg

```
@@ -6,4 +6,4 @@ testpaths = tests
 flake8-max-line-length = 120
 
 [metadata]
-description-file = README.rst
+description_file = README.rst
```

It seems that this is not even a valid field according to https://setuptools.pypa.io/en/latest/userguide/declarative_config.html

This is the closest:



```suggestion
long_description= file: README.rst
```

---

Do you know if we even need this? The description is defined in the `setup.py` as well.
Then I would just remove it from here...